### ISSUE TITLE
Docs feedback: /docs/sources/operations/storage/logs-deletion.md

### ISSUE THREAD

**Issue by Canuteson** [grafana/loki#11361]
Log entry deletion is supported with TSDB shipper, but the documentation still indicates it is only supported with BoltDB Shipper.
```
Log entry deletion is supported only when the BoltDB Shipper is configured for the index store.
```

**Comment by Canuteson** [comment]
I see this is already corrected in the referenced Markdown file, as of 2 months ago. Is there a typical cadence for publishing updated docs?

**Comment by JStickler** [comment]
@Canuteson In this case the updates to the Storage docs are part of a larger effort to remove unused or deprecated code, which won't be released until the next major Loki release (3.0).  Since this doc update is already done, just not released, can we close this issue?

**Comment by Canuteson** [comment]
Yep, sounds good. Thanks!
